test/setup/index.ts

```diff
@@ -37,6 +38,15 @@ export async function removeMonitor(): Promise<void> {
   }
 }
 
+export async function removeLocalContainerRegistry(): Promise<void> {
```

Comment:
This also removes the docker network so the name is not entirely correct, I suggest we split it into two functions, one for each action. Note that currently if `docker rm kind-registry` fails then `docker network prune` won't be called

Reply:
Sounds good